balrog.wsgi

```diff
@@ -1,8 +1,20 @@
 import logging
 from os import path
+from os import getenv
 import site
 import sys
 
+try:
+    import newrelic.agent
```

Comment:
Looks like this is a compiled library. It should get added to the requirements/compiled.txt.